core/include/kernel/dt_driver.h

```diff
@@ -20,14 +20,16 @@
  * DT_DRIVER_UART   UART driver currently designed for console means
  * DT_DRIVER_CLK    Clock controller using generic clock DT bindings
  * DT_DRIVER_RSTCTRL Reset controller using generic reset DT bindings
- * DT_DRIVER_I2C    I2C bus controlle using generic I2C bus DT bindings
+ * DT_DRIVER_I2C    I2C bus controller using generic I2C bus DT bindings
```

Comment:
this fix is not related to GPIO. Could you move it to a specifc commit?

Reply:
Done